### PR TITLE
[docs] adding third party logging and tracing integration documentation for

### PR DESCRIPTION
**Pull request by iamveritas**
adding third party logging and tracing integration documentation for zipkin and deep-mind

### CODE REVIEW

File: docs/01_nodeos/06_logging/10_native_logging/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+content_title: Nodeos Native Logging
+link_text: Nodeos Native Logging
```
Comment (lparisc) [inline docs/01_nodeos/06_logging/10_native_logging/index.md:3]: link_text: Native Logging
(for consistency, as in the other left nav menu items under Nodeos)

File: docs/01_nodeos/06_logging/20_third_party_logging/10_deep_mind_logger.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+DMLOG CREATION_OP ROOT 0
+
+DMLOG RLIMIT_OP ACCOUNT_USAGE UPD {"owner":"eosio","net_usage":{"last_ordinal":1316982371,"value_ex":0,"consumed":0},"cpu_usage":{"last_ordinal":1316982371,"value_ex":24855,"consumed":101},"ram_usage":27083}
+
+DMLOG APPLIED_TRANSACTION 30515 {"id":"308f77bf49ab4ddde74d37c7310c0742e253319d9da57ebe51eb7b35f1ffe174","block_num":30515,"block_time":"2020-11-12T10:13:05.500",...}
+
+DMLOG RLIMIT_OP STATE UPD {"average_block_net_usage":{"last_ordinal":30514,"value_ex":0,"consumed":0},"average_block_cpu_usage":{"last_ordinal":30514,...} \
+DMLOG ACCEPTED_BLOCK 30516 {"block_num":30516,"dpos_proposed_irreversible_blocknum":30516,"dpos_irreversible_blocknum":30515,...
+```
```
Comment (lparisc) [inline docs/01_nodeos/06_logging/20_third_party_logging/10_deep_mind_logger.md:29]: For more clarity It's better to split actual shell commands from output:

For instance:
\`\`\`sh
DMLOG RLIMIT_OP ACCOUNT_USAGE UPD {"owner":"eosio","net_usage":{"last_ordinal":1316982371,"value_ex":0,"consumed":0},"cpu_usage":{"last_ordinal":1316982371,"value_ex":24855,"consumed":101},"ram_usage":27083}
\`\`\`

becomes:

\`\`\`sh
DMLOG RLIMIT_OP ACCOUNT_USAGE UPD {"owner":"eosio","net_usage":
\`\`\`
\`\`\`console (or \`\`\`json if output is JSON as in this case)
{"last_ordinal":1316982371,"value_ex":0,"consumed":0},"cpu_usage":{"last_ordinal":1316982371,"value_ex":24855,"consumed":101},"ram_usage":27083}
\`\`\`

etc...

File: docs/01_nodeos/06_logging/20_third_party_logging/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+content_title: Third-Party Logging And Tracing Integration
+link_text: Third-Party Logging And Tracing Integration
```
Comment (lparisc) [inline docs/01_nodeos/06_logging/20_third_party_logging/index.md:3]: This left nav menu item renders too long (might wrap to next line). Consider:

link_text: Third-Party Logging And Tracing

File: docs/01_nodeos/06_logging/20_third_party_logging/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+## Overview
+
+To stay informed about the overall and detailed performance of your EOSIO-based blockchain node(s), you can make use of the telemetry tools available. EOSIO offers integration with two such telemetry tools:
+
+* [Deep-mind logger](10_deep_mind_logger.md)
+* [Zipkin tracer](20_zipkin_tracer.md)
+
+## Performance Considerations
+
+Many wise people already said it, everything comes with a price. The telemetry tools, when activated, will have a certain impact on your `nodeos` performance which depends on various factors, but nevertheless the performance will be impacted. Therefore it is recommended you use them wisely in those situations when you really need the extra detailed information they provide, and then you turn them off.
```
Comment (lparisc) [inline docs/01_nodeos/06_logging/20_third_party_logging/index.md:15]: The first sentence is a bit too informal - consider removing it (the tradeoff is obvious in the 2nd sentence). Also the 2nd sentence itself is a bit too repetitive. Consider:

The telemetry tools, when activated, will have some impact on your `nodeos` performance depending on various factors.

3rd sentence, consider:
Therefore, it is recommend to enable them only when you need the extra detailed information, then turn them off to regain performance.

File: docs/01_nodeos/06_logging/index.md [hunk 5]
```diff
@@ -1,111 +1,9 @@
 ---
-content_title: Nodeos Logging
+content_title: Nodeos Logging And Tracing
+link_text: Nodeos Logging And Tracing
```
Comment (lparisc) [inline docs/01_nodeos/06_logging/index.md:3]: link_text: Logging And Tracing

File: docs/01_nodeos/06_logging/index.md [hunk 6]
```diff
@@ -1,111 +1,9 @@
...
-    "appenders": [
-        "net"
-    ]
-}
-```
-
-[[info]]
-| The default logging level for all loggers if no `logging.json` is provided is `info`. Each logger can be configured independently in the `logging.json` file.
+* [Nodeos Native Logging](10_native_logging/index.md)
+* [Nodeos Logging And Tracing Third-Party Integration] (20_third_party_logging/index.md)
```
Comment (lparisc) [inline docs/01_nodeos/06_logging/index.md:9]: In 2nd link "Third-Party" got swapped towards the end; also extra space between text and link. Also "Nodeos Third-Party Logging and Tracing" is harder to parse (nodeos is implied). Consider simplifying to:

\* \[Native Logging\](10_native_logging/index.md)
\* \[Third-Party Logging And Tracing\](20_third_party_logging/index.md)

**Review by lparisc — CHANGES_REQUESTED** [review]

File: docs/01_nodeos/06_logging/20_third_party_logging/10_deep_mind_logger.md (hunk 2 above)
Comment (iamveritas) [reply docs/01_nodeos/06_logging/20_third_party_logging/10_deep_mind_logger.md:29]: everything you see here is just shell output from nodeos, no shall command

File: docs/01_nodeos/06_logging/index.md (hunk 6 above)
Comment (iamveritas) [reply docs/01_nodeos/06_logging/index.md:9]: I don't understand any swapping of 'Third-Party' towards the end... can you rephrase that?

File: docs/01_nodeos/06_logging/20_third_party_logging/index.md (hunk 4 above)
Comment (iamveritas) [reply docs/01_nodeos/06_logging/20_third_party_logging/index.md:15]: thank you for your feedback.

File: docs/01_nodeos/06_logging/20_third_party_logging/10_deep_mind_logger.md (hunk 2 above)
Comment (iamveritas) [reply docs/01_nodeos/06_logging/20_third_party_logging/10_deep_mind_logger.md:29]: I have updated the text preceding these examples to be clear that those are coming from nodeos console output.

Comment (lparisc) [reply docs/01_nodeos/06_logging/20_third_party_logging/10_deep_mind_logger.md:29]: Ah got it! In that case it might be wise to rename the codeblock to \`\`\`console (or remove 'sh') since \`\`\`sh is reserved for actual typed shell commands - eventually the docsgen tool might prepend shell prompts $ to each new line in \`\`\`sh codeblocks across the dev portal, which would then render incorrectly in this page.

**Review by lparisc — APPROVED** [review]
I just left a suggestion regarding codeblock usage.
